tools/codestyle/uncrustify_musescore.cfg

```diff
@@ -248,7 +248,7 @@ sp_assign                                = ignore   # ignore/add/remove/force
 sp_cpp_lambda_assign                     = remove   # ignore/add/remove/force
 
 # Add or remove space after the capture specification in C++11 lambda.
-sp_cpp_lambda_square_paren               = ignore   # ignore/add/remove/force
+# sp_cpp_lambda_square_paren               = ignore   # ignore/add/remove/force
```

Comment:
was that an intended change? Seems unrelated to the issue this PR tries to solve

Reply:
See the note in the PR body: 

>Please also note that a line in the config which is not available in the version of Uncrustify in the Ubuntu package repo has been commented out. That shouldn't be a problem, because it was the default value anyway.

Reply:
For now at least. Other she'll scripts don't use tabs though

Reply:
Ah, missed that